en/docs/learn/authentication-session-persistence.md

```diff
@@ -107,10 +107,10 @@ few tasks to clean the data in the database for some given conditions.
 
 #### Task Pool
 
-We are storing above data within a critical path which controls the
-authentication. So we have introduced a pooling mechanism to put the
+On some older releases, the task pool was employed when storing above data within a critical path which controls the
+authentication. Here we have introduced a pooling mechanism to put the
```

Comment:
lets try to avoid using "we"? please check the changes in other places in this doc too